### PR TITLE
🐛 Fix support for Pydantic v2.0, small changes in their final release

### PR DESCRIPTION
**Pull request by Lancetnik**

### CODE REVIEW

**Comment by Lancetnik** [comment]
@tiangolo the job is done! There were some error text and imports changes

**Comment by tiangolo** [comment]
Amazing, thanks! Let me check it all.

**Comment by Lancetnik** [comment]
NP, waiting for fixes release. This thing brokes my [CI](https://github.com/Lancetnik/Propan/actions/runs/5425806226), so I am here and at **polyfactory** too

File: requirements-tests.txt [hunk 1]
```diff
@@ -7,7 +7,7 @@ mypy ==1.3.0
 ruff ==0.0.272
 black == 23.3.0
 httpx >=0.23.0,<0.24.0
-email_validator >=2.0.0,<3.0.0
+email_validator >=2.0.0.post2,<3.0.0
```
Comment (tiangolo) [inline requirements-tests.txt:10]: Is there any reason why this change was needed?

Comment (Lancetnik) [reply requirements-tests.txt:10]: With the environment, installed by default, tests not pass. So, I update email-validator and it fixes a some of them.

Comment (tiangolo) [reply requirements-tests.txt]: Strange, I just manually installed explicitly `2.0.0` to see what was breaking 🤔 

Let's see if reverting works or if I have to put it back.

**Comment by tiangolo** [comment]
Awesome, thank you! 🚀 🎉

**Comment by tiangolo** [comment]
I just published this in FastAPI `0.100.0-beta2`, thanks! 🚀 🍰

**Comment by Lancetnik** [comment]
Thank you for this great project!

**Comment by Lancetnik** [comment]
> I just published this in FastAPI `0.100.0-beta2`, thanks! 🚀 🍰

So, If you still here, can you please take a look and comment my another [PR](https://github.com/tiangolo/fastapi/pull/9630)?
It's even smaller (3 code lines and some tests), but more important for me
